package.json

```diff
@@ -92,8 +102,7 @@
     "all": "npm run lint && npm run flow && npm run test && npm run build && npm run size",
     "build": "npm run clean && npm run build:lib && npm run build:max && npm run build:min && npm run build:tests",
     "build:lib": "babel src --out-dir lib && npm run flow:copy-src",
-    "build:max": "cross-env NODE_ENV=development webpack src/index.js dist/jss.js",
-    "build:min": "cross-env NODE_ENV=production webpack src/index.js dist/jss.min.js",
+    "build:umd": "rollup -c",
```

Comment:
will this generate minified and non-minified version?

Reply:
Yes. I use multiconfig, array of configs.